english/captain/captain.md

```diff
@@ -1 +1,3 @@
-Good Choice.
\ No newline at end of file
+Good Choice.
+
+A man of honour beats a genius billionaire playboy philanthropist. Let's watch the trailer of him fighting iron man [here](https://youtu.be/uVdV-lxRPFo).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.